.github/workflows/changelog-auto.yml

```
@@ -0,0 +1,90 @@
+
+      - name: Setup GitHub CLI
+        run: |
+          sudo apt-get update && sudo apt-get install -y jq gh
+          gh auth setup-git
+          echo "gh CLI installed successfully."
+
+      - name: Fetch CodeRabbit summary
+        id: coderabbit
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.8)

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)